src/test/java/uk/gov/pay/ledger/pact/CaptureSubmittedEventQueueContractTest.java

```diff
@@ -47,7 +48,8 @@ public MessagePact createPaymentCreatedEventPact(MessagePactBuilder builder) {
                 .withResourceExternalId(externalId)
                 .withEventDate(eventDate)
                 .withGatewayAccountId(gatewayAccountId)
-                .withDefaultEventDataForEventType("CAPTURE_SUBMITTED");
+                .withEventType(SalientEventType.CAPTURE_SUBMITTED.toString())
```

Comment:
what has this line got to do with this PR?

Reply:
While fixing the tests (which actually was to rollback some changes) I've noticed that event type is not specified so even though the test is about `CAPTURE_SUBMITTED` event it was actually always `PAYMENT_CREATED`.

Reply:
👍